rules/use-github-discussions/rule.md

```diff
@@ -0,0 +1,48 @@
+---
+type: rule
+archivedreason: 
+title: Do you use Discussions to suggest new ideas?
+guid: 89c61225-1320-4a27-b9de-dafa54591a6f
+uri: use-github-discussions
+created: 2022-01-25T09:31:0.0000000Z
+authors:
+- title: Sam Wagner
+  url: https://ssw.com.au/people/sam-wagner
+related: []
+redirects:
+---
+
+Discussions are a great place to chat with your team and brainstorm new ideas!
+
+If you're thinking of making a Pull Request/Issue to suggest some changes to a document, code or something else. If you're not sure about your suggestions, you could leverage GitHub Discussions to see what people think!
+
+Here's a [great video](https://www.youtube.com/watch?v=bErGYN3Ljz8) by the GitHub team that explains Discussions in full!
+
+`youtube: https://www.youtube.com/embed/bErGYN3Ljz8`
+
+**Video (40sec): What is GitHub Discussions?**
+
+<!--endintro-->
+
+When you're looking to make some changes, chat about the project or or otherwise engage with a project on GitHub. There are some simple steps you can take to choos the right place to engage. They are:
```

Comment:
```suggestion
When you're looking to make some changes, chat about the project or or otherwise engage with a project on GitHub. There are some simple steps you can take to choose the right place to engage. They are:
```